tests/pytorch_pfn_extras_tests/onnx_tests/test_as_output.py

```diff
@@ -95,3 +95,33 @@ def forward(self, x):
     assert 'MatMul_2' in named_nodes
 
     assert len([v.name for v in actual_onnx.graph.output]) == 1
+
+
+def test_as_output_in_scripting():
+
+    class Net(nn.Module):
+        def __init__(self):
+            super(Net, self).__init__()
+            self.conv = nn.Conv2d(1, 6, 3)
+            self.linear = nn.Linear(30, 20, bias=False)
+
+        def forward(self, x, b):
+            h = self.conv(x)
+            if b:  # IF stmt. to check scripted (not traced) IR
+                h = -h
+            h = as_output("h", h)
+            h = self.linear(h)
+            return h
+
+    model = torch.jit.script(Net())
+    x = torch.ones((1, 1, 32, 32))
+    b = torch.tensor(True)
+    output_dir = _helper(model, (x, b), 'as_output', use_pfto=False)
```

Comment:
Seems pfto_exporter dose not support scripting as it calls `torch.jit.trace`. Maybe we should also fix it, but I'm not confident about it as I'm not sure the purpose of pfto_exporter.